TypeScript/webpack.rules.ts

```diff
@@ -28,4 +29,14 @@ export const rules: Required<ModuleOptions>['rules'] = [
       },
     },
   },
+  // {
```

Comment:
Do we need this?

Reply:
Again a note for myself for a later date really.